easybuild/easyblocks/p/parmetis.py

```diff
@@ -167,19 +168,31 @@ def install_step(self):
         # and header files in the Lib directory (capital L). The following symlink are hence created.
         try:
             llibdir = os.path.join(self.installdir, 'Lib')
+            if os.path.lexists(llibdir):
+                os.remove(llibdir)
             os.symlink(libdir, llibdir)
             for f in ['metis.h', 'parmetis.h']:
+                if os.path.lexists(os.path.join(libdir, f)):
+                    os.remove(os.path.join(libdir, f))
```

Comment:
please use the `remove_file` and `symlink` functions from `easybuild.tools.filetools`, and change the code to not use a single-letter variable here:

```python
for header_file in ['metis.h', 'parmetis.h']:
    if os.path.lexists(os.path.join(libdir, header_file)):
        remove_file(os.path.join(libdir, header_file))
        symlink(os.path.join(includedir, header_file), os.path.join(libdir, header_file))

Reply:
Fixed :+1: